### DISCUSSION TITLE
MQTT Device Id

### DISCUSSION THREAD

**Discussion by ghost** [kellerza/sunsynk#285] [Q&A]
Is it possible to send a device id to the MQTT broker? When I restart Sunsynk/Deye Inverter Add-on (multi), the MQTT broker receives a different id and sees it as a new device.

**Comment by kellerza** [comment]
Device IDs are always used and tied to the inverted serial number.

You could try using the (edge/dev) version of the addon, but the device ID mechanisms did not change from what I can recall
